eclair-core/pom.xml

```diff
@@ -270,6 +270,26 @@
             <version>1.5.0</version>
         </dependency>
         <!-- MONITORING -->
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient</artifactId>
+            <version>0.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_hotspot</artifactId>
+            <version>0.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_httpserver</artifactId>
+            <version>0.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_pushgateway</artifactId>
+            <version>0.15.0</version>
+        </dependency>
```

Comment:
Why do you need that? According to [Kamon's doc](https://kamon.io/docs/latest/reporters/prometheus/) you should only need to import:

```
<dependency>
  <groupId>io.kamon</groupId>
  <artifactId>kamon-prometheus_2.13</artifactId>
  <version>2.5.3</version>
</dependency>
```

Reply:
I have included dependencies suggested by [Kamon doc](https://kamon.io/docs/latest/reporters/prometheus/) and as a result, the metrics exposed on port localhost:9095/metrics and we can  change the exposing port in eclair.conf file. 

After this all. Pulling over metrics over HTTP offers a number of advantages but it should not be considered a major point when considering a monitoring system. According to task and discussion I suppose to implement [remote write tunning](https://prometheus.io/docs/practices/remote_write/) right?

As suggested by this documentation [remote_write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) in yaml file

If I am correct then I will add short and crisp code in boot process

Reply:
I think you're still misunderstanding the goal of this feature. What I'd like you to do is setup eclair to be able to export the **existing** metrics (not add new ones!) to a **remote** prometheus endpoint. Nothing more! You should definitely *not* create a prometheus http server inside eclair.

Reply:
Okay got it
But before exporting to remote system. I tried to export it on localhost
and I found some issues maybe due to existing metrics.

it shows 
![down](https://user-images.githubusercontent.com/66783850/171675697-42a5b2b7-ae76-497a-a134-62824ef4d29a.png)

I tried to figure out problem with the help [promtool](https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/)(which is used for testing rules).

and got the output
![output](https://user-images.githubusercontent.com/66783850/171676459-70f66e03-4371-48a8-a9de-146a37fb6df4.png)

due to metric which contains
![port 8080](https://user-images.githubusercontent.com/66783850/171676763-499a9e09-39ed-4bc8-9a37-016bc8b16920.png)

It is better to discuss whole things in meet at Monday,
see you there!